crypto/rand/rand_lib.c

```diff
@@ -25,6 +25,45 @@ static CRYPTO_RWLOCK *rand_meth_lock;
 static const RAND_METHOD *default_RAND_meth;
 static CRYPTO_ONCE rand_init = CRYPTO_ONCE_STATIC_INIT;
 
+#ifdef OPENSSL_RAND_SEED_RDTSC
+/*
+ * Since we get some randomness from the low-order bits of the
+ * high-speec clock, it can help.  But don't return a status since
+ * it's not sufficient to indicate whether or not the seeding was
+ * done.
+ */
+void rand_rdtsc(void)
+{
+    unsigned char c;
+    int i;
+
+    for (i = 0; i < 10; i++) {
+        c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
.5 bytes of course being 4 bit

And it _does_ count toward getting to the seeded successfully value, except when you set it to 0.

And if you say that I shouldn't use the TSC when it's not available, it will just mean to never use it because I don't know at compile time if it's going to be available or not.